src/ShadowMem.cc

```diff
@@ -253,14 +249,15 @@ class ShadowMemImpl : public InstVisitor<ShadowMemImpl> {
 
   // We use std::map to keep keys ordered.  
   using ShadowsMap =
-      std::map<const sea_dsa::Node *, std::map<unsigned, llvm::AllocaInst *>,
+      std::map<const dsa::Node *, std::map<unsigned, llvm::AllocaInst *>,
```

Comment:
we changed namespace from `sea_dsa` to `dsa`?

Reply:
We have the alias `namespace dsa = sea_dsa;`. Sometimes we were using `dsa` and sometimes `sea_dsa`. I'm trying to be consistent and use always `dsa`

Reply:
ok